packages/communicator/vite.config.ts

```diff
@@ -21,7 +21,7 @@ export default defineConfig({
       exclude: ["**/*.test.ts", "vitest.setup.ts"], // Exclude test files from type generation
       include: ["src/**/*.ts", "types/**/*.ts"], // Include your source and global types
       insertTypesEntry: true, // Ensure types are properly exported
-      outDir: "build/types", // Specify where to output the types
```

Comment:
I remember we have discussed this on the slack. Would you also add some comment here?

Reply:
I also wonder what what is the reason so you put now everything into index.d.ts file right?

Reply:
I don't remember the slack thread now but I put the reason in the PR description.